packages/lodestar-params/src/utils.ts

```diff
@@ -13,7 +13,13 @@ export function createIBeaconParams(input: Record<string, unknown>): Partial<IBe
   return params;
 }
 
-export const schema = new Schema({
+export function loadConfigYaml(configYaml: string): Record<string, unknown> {
+  return load(configYaml, {
+    schema: schemaParseAllNumsAsStrings,
+  });
+}
+
+export const schemaParseAllNumsAsStrings = new Schema({
```

Comment:
I would keep schema name as it was because we might gonna need to add more rules for different types